python-package/lightgbm/callback.py

```diff
@@ -258,11 +258,24 @@ def _gt_delta(self, curr_score: float, best_score: float, delta: float) -> bool:
     def _lt_delta(self, curr_score: float, best_score: float, delta: float) -> bool:
         return curr_score < best_score - delta
 
+    def _is_train_set(self, ds_name: str, eval_name: str, train_name: str) -> bool:
+        return (ds_name == "cv_agg" and eval_name == "train") or ds_name == train_name
+
     def _init(self, env: CallbackEnv) -> None:
-        self.enabled = not any(env.params.get(boost_alias, "") == 'dart' for boost_alias
-                               in _ConfigAliases.get("boosting"))
+        is_dart = any(env.params.get(alias, "") == 'dart' for alias in _ConfigAliases.get("boosting"))
+        only_train_set = (
+            len(env.evaluation_result_list) == 1
+            and self._is_train_set(
+                env.evaluation_result_list[0][0],
+                env.evaluation_result_list[0][1].split(" ")[0],
+                env.model._train_data_name)
```

Comment:
```suggestion
                ds_name=env.evaluation_result_list[0][0],
                eval_name=env.evaluation_result_list[0][1].split(" ")[0],
                train_name=env.model._train_data_name)
```

Can you please use keyword arguments here? I think this makes it easier to read this code and understand what it's doing, and that habit prevents mistakes of the form "accidentally passed arguments in the wrong order", which is extra important in cases like this where the arguments have the same type and overlapping ranges of valid values.